docs/01-Welcome-to-FOME.md

```diff
@@ -1,32 +1,25 @@
----
-sidebar_position: 1
-slug: /
----
-
```

Comment:
Root doc/page needs to have a `slug: /`, otherwise we got 404 as a welcome message

```md
---
slug: /
---
```

Reply:
add above important info as comment